cloud/src/test/scala/org/apache/spark/cloud/AzureInstantiationSuite.scala

```diff
@@ -0,0 +1,29 @@
+/*
```

Comment:
Hm, more tests are usually good though if it's just verifying the classpath, it may not be worth it, especially in the name of keeping this simple.